packs/elasticsearch/sensors/count_sensor.py

```diff
@@ -0,0 +1,69 @@
+from st2reactor.sensor.base import PollingSensor
+from elasticsearch import Elasticsearch
+import json
+import time
+
+
+class ElasticsearchCountSensor(PollingSensor):
+
+    def setup(self):
+        self.host = self._config.get('host', None)
```

Comment:
I think you can simply use self.config() as of 1.3 onwards. self._config is deprecated.